docs/changelog.md

```diff
@@ -5,6 +5,23 @@ toc:
 
 # Redocly CLI changelog
 
+
+## 1.0.0-beta.129 (2023-06-26)
+
+### Features
+
+- Added telemetry.
```

Comment:
Can we call this "Added product metrics collection" to be more specific about the purpose?

Reply:
Surely!